src/lib/components/QuestionPaletteModal.svelte.test.ts

```
@@ -116,33 +81,17 @@ describe('QuestionPaletteModal', () => {
-		expect(instructionsTab).toHaveAttribute('aria-controls', 'instructions-panel');
-	});
-
-	it('should apply active tab styling', () => {
-		render(QuestionPaletteModal, { props: defaultProps });
-
-		const paletteTab = screen.getByRole('tab', { name: /question palette/i });
-		expect(paletteTab).toHaveClass('bg-blue-100');
-		expect(paletteTab).toHaveClass('font-bold');
-	});
+	it('should close modal after navigating to a question', async () => {
+		let open = true;
```

ESLint

[error] 'open' is never reassigned. Use 'const' instead.

(prefer-const)